domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidInntektsmelding/ArbeidsforholdInntektsmeldingMangelTjeneste.java

```diff
@@ -118,6 +120,7 @@ public void lagreManuelleArbeidsforhold(BehandlingReferanse behandlingReferanse,
 
     private boolean finnesAvklaringSomGjelderMangel(ArbeidsforholdMangel mangel, List<ArbeidsforholdValg> alleAvklaringer) {
         return alleAvklaringer.stream()
+            .filter(ak -> ArbeidsforholdKomplettVurderingType.FORTSETT_UTEN_INNTEKTSMELDING.equals(ak.getVurdering()))
```

Comment:
Så du ønsker å sjekke om det finnes avklaringer for en arbeidsgiver hvor saksbehandler har vurdert at de ikke trenger inntektsmelding?